backend/views.py

```diff
@@ -78,8 +78,15 @@ def index(request, backend=None, short=None):
         # safe to session
         request.session['backend'] = activeBackend.pk
 
-        # get examples
-        examples = Example.objects.filter(backend=activeBackend)
+        # Get examples, ordered by `sortKey` (examples with empty `sortKey` last)
+        maxSortKey = models.Value("~" * 100)
```

Comment:
Assumes that only ASCII characters are used. Might be good to add that to the comment.